### PR TITLE
Run executables from /bin instead of /app/bin on development

### LINKED ISSUE #770: Allow for dev to use the same bin/ directory
Instead of having to use 2 different directories for binaries for production and dev (`bin/` and `app/bin/` respectively).  We should be able to consolidate into using the same `bin/` for both.

### PR DESCRIPTION
**Pull request by matheusd**
Fix #770

### CODE REVIEW

**Comment by matheusd** [comment]
Latest push also fixes #691

**Comment by alexlyp** [comment]
Strange on sync/blockchain DL it displayed what appeared to be the correct required block height but then when it got to what appeared to be the estimated height it moved on (156K, 149K respectively)

**Comment by matheusd** [comment]
Should be all right now. Bug was a misplaced closure.

File: app/components/views/GetStartedPage/DaemonLoading/index.js [hunk 1]
```diff
@@ -1,38 +1,49 @@
...
+    this.mounted = true;
+    this.timeoutId = this.props.setTimeout(() => {
+      if (this.mounted) {
+        this.setState({ showLongWaitMessage: true });
+      }
+    }, 2000);
+    const neededBlocksInterval = this.props.network === "mainnet"
+      ? 5 * 60 * 1000
+      : 2 * 60 * 1000;
+    this.props.setInterval(this.props.determineNeededBlocks, neededBlocksInterval);
```
Comment (alexlyp) [inline app/components/views/GetStartedPage/DaemonLoading/index.js]: Can you explain why you set and interval here based on network target block time?  Not sure it's necessary.  Really just need it once on startup

Comment (matheusd) [reply app/components/views/GetStartedPage/DaemonLoading/index.js]: During initial block download (on a first time run with a clean block database) the initial sync may take minutes/hours. During that time the block height will change, so until the daemon is completely synced to the network it should try and find the needed block height.

I understand that the difference may be small (I added the interval check mainly for completeness), so if you feel the block difference is small we can take it off.

Comment (alexlyp) [reply app/components/views/GetStartedPage/DaemonLoading/index.js]: oh very nice!  didn't think about that situation.  nope that's totally understandable now.  nice work!

**Pull request merged**